tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,242 @@
+
+Action sets
+===========
+
+.. note::
+  Before we dive in, you will see the term XR runtime used throughout this document. With XR runtime we mean the software that is controlling and interacting with the AR or VR headset and exposed to us through an API such as OpenXR. So for Steam this is SteamVR, for Meta on desktop this is the Oculus Client (including when using Quest link), for Meta on Quest this is the Quests native OpenXR client, on Linux this could be Monado, etc.
+
+The action map allows us to organize our actions in sets. Each set can be enabled or disabled on its own.
+
+The concept here is that you could have different sets that provide bindings in different scenarios. You could have a set ``Character control`` for when you're walking around, ``Vehicle control`` for when you're operating a vehicle, ``Menu`` for when a menu is open, etc.
+Only the action set applicable to the current state of your game/application can then be enabled.
+
+This is especially important if you wish to bind the same input on a controller to a different action. For instance:
+
+  * in your ``Character control`` set you may have an action ``Jump``, 
+  * in your ``Vehicle control`` set you may have an action ``Accelerate``, 
+  * in your ``Menu`` set you may have an action ``Select``.
+
+All are bound to the trigger on your controller.
+
+OpenXR will only bind an input or output to a single action. If the same input or output is bound to multiple actions the one in the active action set with the highest priority will be the one updated/used. So in our above example it will thus be important that only one action set is active.
+
+For your first XR game/application we highly recommend starting with just a single action set and not over-engineer things.
+
+For our walk through in this document we will thus create a single action set called ``my_first_action_set``, we do this by pressing the ``Add action set`` button:
+
+.. image:: img/xr_my_first_action_set.webp
+
+The columns in our table are as follows:
+
+.. list-table::
+  :class: wrap-normal
+  :width: 100%
+  :widths: 7 23 70
+  :header-rows: 1
+
+  * - Col
+    - Value
+    - Description
+  * - 1
+    - my_first_action_set
+    - This is the internal name of the action set. OpenXR doesn't specify specific restrictions on this name other then size, however some XR runtimes will not like spaces or special characters.
+  * - 2
+    - My first action set
+    - This is a human-readable name for the action set. Some XR runtimes will display this name to the end user, for example in configuration dialogs.
+  * - 3
+    - 0
+    - This is the priority of the action set. If multiple active action sets have actions bound to the same controllers inputs or outputs, the action set with the highest priority value will determine the action that is updated.
```

Comment:
What occurs when multiple action sets have the same priority? 
Might be worth clarifying as a `Note`.

Reply:
I think that is undefined, same as having multiple actions in one action set bound to the same input.

Reply:
Ok, I've added a note to the bottom of this section giving some clarification here. I underline as with my discussion with @rsjtdrjgfuzkfg that I'm unsure of this as its not well defined in the specification. I've asked questions but not yet received suitable answers, I also get the feeling this may be in flux.

Dirk, I also added a small remark to this note about the haptic bindings. Again we simply don't know for sure what is allowed and I wouldn't be surprised if different runtimes do different things here.